README.md

```diff
@@ -18,7 +18,9 @@ Sortable UI primitives for Ember.
 |----------------|-------|------|
 | 3.0.0          | 3.24+ | 12+  |
 | 4.0.0          | 3.24+ | 14+  |
-| 5.0.0          | 3.28+ | 14+  |
+| 5.0.0          | 3.28+ | n/a[^1]  |
+
+[^1]: Node is not relevant for v2 addons. As of v5.0.0, ember-sortable is a v2 addon. V2 addons don't have to be for browser-only contexts, but ember-sortable is -- so node is not relevant anymore. This is different from v1 addons, which are not browser libraries, but node programs that happen emit browser code to the consuming app at app-build time.
```

Comment:
This is an important distinction for v2 addons that may be a bit different from what folks are used to